style.css

```diff
@@ -1,101 +1,93 @@
-body,
-h1,
-h2,
-p {
-  margin: 0;
-  color: #091e42;
-}
-
-.nav-bar {
-  display: flex;
-  background-color: #3c3a39;
-  color: #fff;
-  height: 5.77em;
-  align-items: center;
-  padding-right: 1.26em;
-}
-
-.logoname {
-  font-family: "Inter", sans-serif;
-}
-
-.nav-bar a {
-  color: #fff;
-  font-size: 1.26em;
-  flex: 1;
-  display: flex;
-  margin: 0 2rem;
-  text-decoration: none;
-  font-family: "inter", sans-serif;
-}
-
-#menu {
-  justify-content: flex-end;
-  width: 1.125rem;
-  border: none;
-  background-color: transparent;
-  background-image: url(../images/menu.svg);
-  background-repeat: no-repeat;
-  background-position: center;
-}
-
-a.button,
-button {
-  padding: 0.76rem;
-  color: #fff;
-  background: none;
-  text-align: center;
-}
-
-.mobile-view {
-  width: 1.125rem;
-  border: none;
-  background-color: transparent;
-  background-image: url(../images/menu.svg);
-  background-repeat: no-repeat;
-  background-position: center;
-}
-
-.first-sec {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  padding: 0 11%;
-  row-gap: 0.8rem;
-  background-color: #1c1a19;
-  background-image: url(../images/background-mobile.png);
-  background-repeat: no-repeat;
-  background-position: bottom right;
-  background-size: auto 15.626em;
-  height: 38rem;
-  color: #fff;
-}
-
-.first-sec h1 {
-  color: #e05304;
-  font-size: 3.6rem;
-  font-family: "Crete Round", serif;
-}
-
-.first-sec h2 {
-  color: #fff;
-  font-size: 1.5rem;
-  line-height: 2.1rem;
-  font-weight: 700;
-  font-family: "Roboto", sans-serif;
-}
-
-.first-sec p {
-  color: #fff;
-  font-size: 1.2rem;
-  font-family: "Poppins", sans-serif;
-  line-height: 1.6rem;
-}
-
-.social-media-icons {
-  display: flex;
-  gap: 1.2rem;
-  padding: 0;
-  margin: 0;
-  list-style: none;
-}
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+main {
+  background-image: url("header-shapes.png");
+  background-repeat: no-repeat;
+}
+
+header {
+  position: fixed;
+  background-color: #fff;
+  top: 0;
+  width: 375px;
+  height: 90px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header a {
+  margin-top: 58px;
+  margin-left: 24px;
```

Comment:
![image](https://user-images.githubusercontent.com/61361037/212132050-419c01c3-9179-4fab-86c1-b5a4b7be5e21.png)

Kindly remove `margin-top` and `margin-left` for the individual elements of header. It is dragging the logo down and creating issue in alignment.
Add the `margin-top` and `margin-left` to the header tag. That would give equal margin to both the elements.